homeassistant/const.py

```diff
@@ -156,6 +156,7 @@
 CONF_SCENE = "scene"
 CONF_SELECTOR = "selector"
 CONF_SENDER = "sender"
+CONF_SENSOR = "sensor"
```

Comment:
The number of occurrences of `CONF_SENSOR` in our codebase is really low. I'm not sure if we should move this up into our central constants file.